scrape_linkedin/CompanyScraper.py

```diff
@@ -13,13 +13,30 @@
 
 
 class CompanyScraper(Scraper):
-    def scrape(self, company, overview=True, jobs=False, life=False, insights=False):
-        self.url = 'https://www.linkedin.com/company/{}'.format(company)
+
+    def scrape(self,
+               company,
+               org_type="company",
```

Comment:
`org_type` is used only to generate a URL, but LinkedIn seems to automatically redirect the URL in the case you use `/company/school-id`. For example, https://www.linkedin.com/company/harvard-university works (otherwise, your example in `people-to-csv.py` would break.

I think this should be removed as an option for now.